lib/platform/gitlab/index.ts

```diff
@@ -442,33 +436,35 @@ export async function getPrList(): Promise<Pr[]> {
 }
 
 async function tryPrAutomerge(
-  pr: number,
+  iid: number,
   platformOptions: PlatformPrOptions
 ): Promise<void> {
   if (platformOptions?.gitLabAutomerge) {
     try {
       const desiredStatus = 'can_be_merged';
       const retryTimes = 5;
 
+      let mr: GitLabMergeRequest;
       // Check for correct merge request status before setting `merge_when_pipeline_succeeds` to  `true`.
       for (let attempt = 1; attempt <= retryTimes; attempt += 1) {
-        const { body } = await gitlabApi.getJson<{
-          merge_status: string;
-          pipeline: string;
-        }>(`projects/${config.repository}/merge_requests/${pr}`);
+        const { body } = await gitlabApi.getJson<GitLabMergeRequest>(
+          `projects/${config.repository}/merge_requests/${iid}`
+        );
         // Only continue if the merge request can be merged and has a pipeline.
         if (body.merge_status === desiredStatus && body.pipeline !== null) {
+          mr = body;
           break;
         }
         await delay(500 * attempt);
       }
 
       await gitlabApi.putJson(
-        `projects/${config.repository}/merge_requests/${pr}/merge`,
+        `projects/${config.repository}/merge_requests/${iid}/merge`,
```

Comment:
use `updateMR` helper function

Reply:
this one is a `/merge` not an update so I don't think that helper will work?

Reply:
ups, you are right, so leave as is.